low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -17,17 +17,18 @@
 
 
 import time
-import re
 import errno
-from framework.utils.ipmi_client import IpmiFactory
-from framework.base import sspl_constants
-
 import psutil
 from pathlib import Path
+
+from framework.utils.ipmi_client import IpmiFactory
+from framework.base import sspl_constants
```

Comment:
we are importing required data from sspl_constants on line 29, you can remove this line.

Reply:
Done. 👍